app/frontend/src/features/profile/profilePage/Overview.tsx

```diff
@@ -0,0 +1,57 @@
+import { Card, makeStyles } from "@material-ui/core";
+import React from "react";
+
+import Avatar from "../../../components/Avatar";
+import BarWithHelp from "../../../components/Bar/BarWithHelp";
+import Divider from "../../../components/Divider";
+import { CouchIcon, LocationIcon } from "../../../components/Icons";
+import IconText from "../../../components/IconText";
+import LabelAndText from "../../../components/LabelAndText";
+import { HostingStatus, MeetupStatus } from "../../../pb/api_pb";
+import { COMMUNITY_STANDING, COMMUNITY_STANDING_DESCRIPTION, LAST_ACTIVE, REFERENCES, RESPONSE_RATE, RESPONSE_TIME, VERIFICATION_SCORE, VERIFICATION_SCORE_DESCRIPTION } from "../../constants";
+import useCurrentUser from "../../userQueries/useCurrentUser";
+import { hostingStatusLabels, meetupStatusLabels } from "../constants";
+
+const useStyles = makeStyles((theme) => ({
+  card: {
+    padding: 16,
+    width: '30%',
+  },
+  grow: {
+    paddingTop: "100%",
+  },
+  info: {
+    marginTop: theme.spacing(0.5),
+  }
+}));
+
+export default function Overview() {
+  const classes = useStyles();
+  const { data: user } = useCurrentUser();
+
+  return (
+    <Card className={classes.card}>
+      <Avatar user={user} className={classes.grow} />
+      <h1>{user?.name}</h1>
+      <h4>{user?.city}</h4>
+      <Divider />
+      <IconText icon={CouchIcon}
+                text={hostingStatusLabels[user?.hostingStatus || HostingStatus.HOSTING_STATUS_UNKNOWN]} />
+      <IconText icon={LocationIcon}
+                text={meetupStatusLabels[user?.meetupStatus || MeetupStatus.MEETUP_STATUS_UNKNOWN]} />
+      <Divider />
+      <BarWithHelp value={user?.communityStanding || 0} 
+                   label={COMMUNITY_STANDING}
+                   description={COMMUNITY_STANDING_DESCRIPTION} />
+      <BarWithHelp value={user?.verification || 0}
+                   label={VERIFICATION_SCORE}
+                   description={VERIFICATION_SCORE_DESCRIPTION} />
+      <div className={classes.info}>
+        <LabelAndText label={REFERENCES} text={`${user?.numReferences || 0}`}></LabelAndText>
+        <LabelAndText label={LAST_ACTIVE} text={"figure out date" || "Unknown"}></LabelAndText>
```

Comment:
```suggestion
        <LabelAndText label={LAST_ACTIVE} text={user?.lastActive ? `Last active ${timeAgo(timestamp2Date(user.lastActive))}` : "Unknown"}></LabelAndText>
```
Or something like that. (Those functions are in `src/utils/` I think.)